Controller/FolderController.php

```diff
@@ -26,22 +25,10 @@ class FolderController extends Controller
      * Displays the sub folder if it exists
      *
      * @param \eZ\Publish\API\Repository\Values\Content\Location $location of a folder
-     * @throws NotFoundHttpException $location is flagged as invisible
      * @return \Symfony\Component\HttpFoundation\Response
      */
-    public function showFolderListAsideViewAction( Location $location )
+    public function showFolderListAsideViewAction( Location $location, $viewType, $layout = false, array $params = array() )
```

Comment:
Some questions:
Now this name is not really correct anymore is it?
Is it so general the two methods could be merged?
and where is 'aside' specified now btw, or?